<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="yesChange">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>40</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="noChange">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="yesLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Решили перевыбрать дверь</string>
    </property>
   </widget>
   <widget class="QLabel" name="noLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Решили не перевыбирать дверь</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="experiment">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>39</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Эксперименты</string>
    </property>
    <widget class="QRadioButton" name="plusOne">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1 эксперимент</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="plusFive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>5 экспериментов</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="plusTen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10 экспериментов</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="plusFifty">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>50 экспериментов</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="plusFifty_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100 экспериментов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paradocs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Провести</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="yesStatistic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="noStatistic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Statistic">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
